_posts/2022-07-25-release-kubernetes-operator-1.1.0.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title:  "Apache Flink Kubernetes Operator 1.1.0 Release Announcement"
+subtitle: "Lifecycle management for Apache Flink deployments using native Kubernetes tooling"
+date: 2022-07-25T08:00:00.000Z
+categories: news
+authors:
+- gyfora:
+  name: "Gyula Fora"
+  twitter: "GyulaFora"
+- matyas:
+  name: "Matyas Orhidi"
+
+---
+
+The community has continued to work hard on improving the Flink Kubernetes Operator capabilities since our [first production ready release](https://flink.apache.org/news/2022/06/05/release-kubernetes-operator-1.0.0.html) we launched about two months ago.
+
+With the release of Flink Kubernetes Operator 1.1.0 we are proud to announce a number of exciting new features that improves the overall experience of managing Flink resources and the operator itself in production environments.
+
+## Release Highlights
+
+A non-exhaustive list of some of the more exciting features added in the release:
+
+ * Kubernetes Events on application and job state changes
+ * New operator metrics
+ * Unified and more robust reconciliation flow
+ * Periodic savepoint triggering
+ * Custom Flink Resource Listeners
+ * Dynamic watched namespaces
+ * New built-in examples for submitting Flink SQL and Python jobs
+ * Experimental autoscaling support
+
+### Kubernetes Events for Application and Job State Changes
+
+The operator now emits native Kubernetes Events on relevant Flink Deployment and Job changes. This includes status changes, custom resource specification changes, deployment failures, etc.
+
+```
+Events:
+  Type    Reason         Age   From                  Message
+  ----    ------         ----  ----                  -------
+  Normal  Submit         53m   JobManagerDeployment  Starting deployment
+  Normal  StatusChanged  52m   Job                   Job status changed from RECONCILING to CREATED
+  Normal  StatusChanged  52m   Job                   Job status changed from CREATED to RUNNING
+```
+
+### New Operator Metrics
+
+The first version of the operator only came with basic system level metrics to monitor the JVM process.
+
+In 1.1.0 we have introduced a wide range of additional metrics related to lifecycle-management, Kubernetes API server access and the Java Operator SDK framework the operator itself is built on. These metrics allow operator administrators to get a comprehensive view of what’s happening in the environment.
+
+For details check https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/metrics-logging/#metrics.
```

Comment:
nit: check the list of [supported metrics](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/metrics-logging/#metrics)